src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationProvider.cs

```diff
@@ -521,10 +532,20 @@ private async Task InitializeAsync(bool ignoreFailures, IEnumerable<Configuratio
                     adapter.InvalidateCache();
                 }
 
+<<<<<<< HEAD

```

Comment:
Looks like there were merge conflicts.

Reply:
Sorry not done yet, needed to push to remote real quick

Reply:
Sorry about that, fixed now, tests are still in progress